stix/idl/demo/demo_imaging.pro

```diff
@@ -22,28 +22,42 @@ data_folder = getenv('SSW_STIX') + '/idl/processing/imaging/data/'
 
 path_sci_file = data_folder + 'solo_L1_stix-sci-xray-l1-1178428688_20200607T213708-20200607T215208_V01.fits' ; Path of the science L1 fits file
 path_bkg_file = data_folder + 'solo_L1_stix-sci-xray-l1-1178451984_20200607T225959-20200607T235900_V01.fits' ; Path of the background L1 fits file
-time_range = ['7-Jun-2020 21:39:00', '7-Jun-2020 21:42:49'] ; Time range to consider
-energy_range = [6,10]       ; Energy range to consider (keV)
-mapcenter = [-1625, -700] ; Coordinates of the center of the map to reconstruct (quasi-heliocentric)
-xy_flare = mapcenter      ; Location of the map (quasi-heliocentric). Needed for the visibility phase calibration
+; Path of the auxiliary L2 fits file (available at "http://pub099.cs.technik.fhnw.ch/fits/L2/")
```

Comment:
minor point but might be nicer to use the datarchive URL rather than pub099 as it seems a bit friendlier

Reply:
What is the correct url to be used? This [link](http://pub099.cs.technik.fhnw.ch/fits/L2/) is the only one I know containing the L2 fits files

Reply:
It has now been set up that dataarchive.stix.i4ds.net points to the same place as pub099.cs.technik.fhnw.ch so to get to the files you can do http://dataarchive.stix.i4ds.net/fits/L2/. It's all going to the same place so it's not an issue I just think the new ones are nicer and clearer for the users. The same as pub023 can now be found at datacenter.stix.i4ds.net

Reply:
Ok, thanks! I'll change it then